### ISSUE TITLE
Default font size is too small

### ISSUE THREAD

**Issue by necessarily-equal** [Unvanquished/Unvanquished#1819]
I've found the font size RML-UI gives by default for 1em is pretty small, and I find the UI to be better looking with that patch

```diff
diff --git a/ui/shared/window.rml b/ui/shared/window.rml
index 88c5520..dada0a0 100644
--- a/ui/shared/window.rml
+++ b/ui/shared/window.rml
@@ -8,10 +8,11 @@
                window {
                        display: block;
                        z-index: 5;
                        overflow: auto;
                        padding: 1.5em;
+                       font-size: 1.2em;
```

However this doesn't change the size everywhere, for example at several places in-game and on the in-game menu.

Would it be possible to change the default font size so that it applies to everyhing? @slipher would you know maybe?

**Comment by slipher** [comment]
I don't really understand how the 'em' unit works. A lot of stuff randomly changed size upon switching to RmlUi.

There's a new `dp` unit in RmlUi which is a sort of absolute size, except that it scales with our global scale factor unlike `px`. Maybe that would be better to use in the future.

**Comment by illwieckz** [comment]
Do we have `%` unit for font size in RmlUi? If yes it would be good to turn every font size into percentage-based font size and just set an hardcoded font size on `body`.

**Comment by necessarily-equal** [comment]
Yes we have relative unit font-size, and I'd say we already use them 
everywhere as far as font sizes are involved.

(The usual unit is the em, for the sake of the learning here's an 
explaination: « The em and ex units depend on the font and may be 
different for each element in the document. The em is simply the font 
size. In an element with a 2in font, 1em thus means 2in. Expressing 
sizes, such as margins and paddings, in em means they are related to 
the font size, and if the user has a big font (e.g., on a big screen) 
or a small font (e.g., on a handheld device), the sizes will be in 
proportion. Declarations such as text-indent: 1.5em and margin: 1em are 
extremely common in CSS. »)
